package.json

```diff
@@ -4,7 +4,7 @@
   "description": "inline markups to HTML: SVG, MathML, etc.",
   "main": "index.js",
   "scripts": {
-    "test": "mocha ./test/index.spec.js"
+    "test": "node_modules/.bin/mocha ./test/index.spec.js"
```

Comment:
when we using `scripts`, the `node_modules/.bin` is default added to PATH, so we can use mocha only.